src/services/transaction-cron.ts

```diff
@@ -79,7 +79,7 @@ export default class TransactionCron {
       });
       let startBlockNumber = 0;
       if (latestTransaction) {
-        startBlockNumber = Number(latestTransaction.destinationBlockNumber);
+        startBlockNumber = Number(latestTransaction.destinationBlockNumber) + 1;
```

Comment:
Why only on Ethereum Receive? Why not also on Ethereum Send?
cc: @rac-sri @0xSasaPrsic

Reply:
It's this https://github.com/availproject/bridge-ui-indexer/pull/18/files#diff-43f55536c98246e0037c26dbb4e658ac2c6d8f37c734e847c35fd480d1fd218dR300-R302